README.md

```diff
@@ -17,20 +17,19 @@ golang libs.  This may prove to not be ideal, and may be changed over time, but
 
 ## Logging and Debug
 
-By default the library does not provide any logging, but provides an error message that includes any messages from
-iscsiadm as well as exit-codes.  In the event that you need to debug the library, we provide a function:
+By default the library uses klog and structured logging to produce InfoS and ErrorS log entries. A caller can replace
+the default io.Writer with their own by using the provided function:
```

Comment:
This documentation is out-dated, right?

`EnableDebugLogging` no longer exists.

Reply:
Yes, corrected to match current code changes.